README.md

```diff
@@ -2,4 +2,42 @@
 
 [![Build Status](https://travis-ci.org/coreos/jetcd.svg?branch=master)](https://travis-ci.org/coreos/jetcd)
 
-Java client for etcd v3.
+jetcd is a Java client library for the awesome [etcd](https://coreos.com/etcd/), a highly available distributed key-value store.
+
+### Etcd version support
+
+jetcd only supports newly etcd v3, for any v2 support, you can use Java bindings list [here](https://github.com/coreos/etcd/blob/master/Documentation/libraries-and-tools.md).
+
+### Download
+
+#### Maven
+
+TODO
+
+#### Gradle
+
+TODO
+
+### Usage
+
+Check out tests for more details about how to use this library, but here's a quick code example:
+
+```
+EtcdClient client = EtcdClientBuilder.newBuilder().endpoints("http://localhost:2379").build();
+EtcdKV kvClient = client.getKVClient();
+
+ByteString key = ByteString.copyFrom("test_key", "UTF-8");
+ByteString value = ByteString.copyFrom("test_value", "UTF-8");
+
+// put the key-value
+kvClient.put(key, value).get();
+// get the value
+ListenableFuture<RangeResponse> getFeature = kvClient.get(key);
+RangeResponse response = getFeature.get();
+assertEquals(response.getKvsCount(), 1);
+assertEquals(response.getKvs(0).getValue().toStringUtf8(), "test_value");
+// delete the key
+kvClient.delete(key).get()
+```
+
+For full etcd v3 API, plesase refer to [API_Reference](https://github.com/coreos/etcd/blob/master/Documentation/dev-guide/api_reference_v3.md).
```

Comment:
Not sure if this is the correct place for this.  I saw this pull request after trying to figure out how to access/use the `KVClient`.  If the usage you are writing here is correct then should `getKVClient `as well as the other get methods on the `EtcdClient `class be `public `instead of `protected`?